app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -85,9 +88,38 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
     binding.footerViewModel = getFooterViewModel()
     binding.executePendingBindings()
 
+    setIfDeveloperOptionsMenuItemListener()
+
     return binding.root
   }
 
+  private fun setIfDeveloperOptionsMenuItemListener() {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
Why is it wrapped for N and above only? And what about below N ?

Reply:
The optional binding feature is supported only for N and above. As this is meant for developers only, I believe it won't be much trouble. WDYT?

@BenHenning @Sarthak2601 what are your thoughts on this?

Reply:
Since this is dev-only feature, I don't think there should be much concern on what happens below N. That being said, @rt4914 is there any specific use case that we'd want to test below android version N?

Reply:
Since this is dev-only feature, I don't think there should be much concern on what happens below N. That being said, @rt4914 is there any specific use case that we'd want to test below android version N?

Reply:
@YashKr01 @Sarthak2601 Does this mean that the feature (Developer Options) will be available to only devices with above N version? If so, we should avoid this and make it accessible by all.

Reason: There might be some contributor who is using an old android phone and does not have emulator strong hardware so for them it won't be accessible. Considering we can make it easily accessible to all developers we should prioritise that.

Reply:
Hmm, seems reasonable. The use of optional is such that it can't be used below android version N but we can achieve this functionality if we have an optional class implementation of our own -- [reference blog](https://medium.com/@coreflodev/use-optional-t-in-android-today-a3f7021e3f7a). @rt4914 @yashraj-01 WDYT? 

Note: this is subject to @BenHenning's approval.

Reply:
@Sarthak2601 yes we can use Optional. Lets wait for @BenHenning 's suggestion too.

Reply:
@rt4914 @Sarthak2601 I have filed an issue for this. PTAL #3383